php-malware-finder/whitelist.md

```diff
@@ -45,7 +45,16 @@ See `whitelist.yar` and the includes in `whitelists/` for instructions on adding
 * 7.3.0
 * 7.4.0
 * 7.4.2
```

Comment:
Just curious why we're not whitelisting the [7.4.1 version](https://github.com/guzzle/guzzle/releases/tag/7.4.1)?